src/misc/inc/MultiDimensionalIndexing.h

```diff
@@ -22,21 +22,21 @@
 //
 //-----------------------------------------------------------------------el-
 
-#ifndef UQ_INTERPOLATION_SURROGATE_HELPER_H
-#define UQ_INTERPOLATION_SURROGATE_HELPER_H
+#ifndef UQ_MULTI_DIMENSIONAL_INDEXING_H
+#define UQ_MULTI_DIMENSIONAL_INDEXING_H
 
 // C++
 #include <vector>
 
 namespace QUESO
 {
-  class InterpolationSurrogateHelper
+  class MultiDimensionalIndexing
```

Comment:
Are you concerned about this breaking older code?  If so, you might consider a simple `typedef` for a release or two before changing the name.